### ISSUE TITLE
Add unit tests for CLI commands

### ISSUE THREAD

**Issue by MakMuftic** [ChainSafe/chainbridge-core#216]
Based on tests added by @mpetrun5 in [this](https://github.com/ChainSafe/chainbridge-core/pull/215/files) PR - we should do the same for all our CLI commands.

This would test our **flag validation/processing logic, flag binding, and also command execution function**.

## Implementation details

A sample of these unit tests can be found in PR mentioned previously.

Before we start implementing this issue - first we should decide if we want to: 
- define a global interface for each contract (this would hold all available methods for the specific contract)
- for each CLI command define a specialized interface that only defines a subset of methods being used in that command (this approach was used in previosuly mentioned PR) 

## Testing details

Make sure to check each component is working properly when writing tests: validation function, processing function, flag binding, command execution.

## Acceptance Criteria

- [ ] add unit tests for each CLI command
- [ ] test all previously mentioned parts for each CLI command

**Comment by itsbobbyzzz168** [comment]
No longer relevant. Closing
